bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/protocol/SBMessage.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2021 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.telenot.internal.protocol;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link SBMessage} class represents a parsed SB message.
+ * *
+ * 
+ * @author Ronny Grun - Initial contribution
+ */
+@NonNullByDefault
+public class SBMessage extends TelenotMessage {
+
+    /** Address number */
+    public final int address;
+
+    public final int disarmed;
+    public final int internallyArmed;
+    public final int externallyArmed;
+    public final int alarm;
+    public final int malfunction;
+    public final int readyToArmInternally;
+    public final int readyToArmExternally;
+    public final int statusInternalSignalHorn;
+
+    public SBMessage(String message) throws IllegalArgumentException {
```

Comment:
I would prefer a check exception here (e.g.`TelenotMessageException extends Exception`) because the compiler takes care of checking if proper exception handling is implemented. Runtime exceptions like `IllegalArgumentException` should only be used if necessary (e.g. to escape a lambda).